src/admin/api.js

```diff
@@ -370,6 +456,8 @@ async function createFxpRateForCurrencyChannel(endpoint, currencyPair, rateDetai
         startTime: rateDetails.startTime,
         endTime: rateDetails.endTime,
         reuse: rateDetails.reuse,
+        forexProviderInfo: rateDetails.forexProviderInfo
+          || getForexProviderInfo(FOREX_PROVIDERS.CITI),
```

Comment:
How do you get all of the info necessary statically? A lot of it depends on the caller.

Reply:
It was missing the required parameters, I updated it so it passes the `currencyPair` and `rateDetails`.